app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -664,6 +664,8 @@ class PatientSummaryScreen :
       excludeTarget(R.id.newBPItemContainer, true)
       excludeTarget(R.id.bloodSugarItemContainer, true)
       excludeTarget(R.id.drugsSummaryContainer, true)
+      // We are doing this to wait for the router transitions to be done before we start this.
+      startDelay = 500
```

Comment:
Not sure if its a good idea to set hardcoded delay? WDYT?

Reply:
This is an animation-delay, it's normal mostly? Also, this is not the final implementation of this feature. It's just for field study, in the final version when we pick up, we have to revisit this animation and how it should work. 

So, I am guessing this should be fine for now. WDYT?

Reply:
or do you mean to have this in the resources? That I can do.

Reply:
No No, not the resources. If we're going to anyway revisit then its fine.